docs/FAQ.rst

```diff
@@ -356,6 +360,24 @@ If you are behind a corporate proxy, make sure you have ``proxy type`` and
 ``aesm proxy`` lines set in ``/etc/aesmd.conf`` .
 This file may be overwritten if you reinstall Intel SGX SDK.
 
+How do I know I am in Intel SGX Hardware Mode?
+----------------------------------------------
+If you set ``SGX_MODE=HW`` in your environment and setup Intel SGX correctly,
+Avalon will startup in Intel SGX Hardware Mode.
+You know you are in Intel SGX Hardware Mode (SGX_MODE=HW) when you see messages
+similar to this in the Avalon Enclave Manager output at startup
+(the SPID and IAS API keys are examples and not active):
+
+
+   .. code:: none
+
+       INFO avalon_enclave_manager.ias_client] IAS settings:
+       INFO avalon_enclave_manager.ias_client] SPID: BA5FDA588B8EA36B203955D38FA6B675
+       INFO avalon_enclave_manager.ias_client] IAS ApiKey: 8730f63c63da49258dd37ecf318997ed
+
+Dan
```

Comment:
Is this a typo or did you actually intend.

Reply:
@rranjan3 Fixed cut and paste error. Thanks.